python/cuml/test/test_pickle.py

```diff
@@ -597,3 +598,29 @@ def assert_model(pickled_model, X):
         assert state["_fit_status_"] == 0
 
     pickle_save_load(tmpdir, create_mod, assert_model)
+
+
+def test_small_rf(tmpdir):
```

Comment:
The test can be updated to pass `'RandomForestClassifier'` as the `key` instead of using the import statement.

Reply:
Done.

Reply:
Thank you!